### DISCUSSION TITLE
Undefined variable $applyStateBindingModifiers

### DISCUSSION THREAD

**Discussion by rahat1994** [filamentphp/filament#8250] [Help]
I am trying to build a filament component/Asynchronous Alpine.js component and pass state to the component but facing this issue the documentation is not clear to me.

I am trying to create a sortable component.

can anyone please help?

filament component file:

`    <div
        x-ignore
        ax-load
        x-load-js="[@js(\Filament\Support\Facades\FilamentAsset::getScriptSrc('sortable-js'))]"
        ax-load-src="{{ \Filament\Support\Facades\FilamentAsset::getAlpineComponentSrc('curriculumns-widget') }}"
        x-data='CurriculumnsWidget({
            state: $wire.{{ $applyStateBindingModifiers("\$entangle('{$statePath}')") }},
        })' x-cloak>`

2. Alpinejs component

`    export default function CurriculumnsWidget({
            state,
        }) {
            return {
                state,
                init: () => {
                    console.log(this.$refs)

            setTimeout(() => {
                const elements = this.$refs.items.querySelectorAll(
                    '.curriculum_modules');
                elements.forEach((ele, i) => {
                    console.log(ele);
                    Sortable.create(ele, {
                        group: 'curriculum_modules',
                        onEnd: (event) => {
                            let list = Alpine.raw(this.list);

                            let origin_module_index = Number(event
                                .from.id.split('_')[3]) - 1;
                            let destination_module_index = Number(
                                event.to.id.split('_')[3]) - 1;

                            let moved_module = list[
                                origin_module_index].modules
                                .splice(event.oldIndex, 1)[0]
                            list[destination_module_index].modules
                                .splice(event.newIndex, 0,
                                    moved_module)
                        },
                    });
                });

                Sortable.create(this.$refs.items, {
                    handle: '.handle',
                    onEnd: (event) => {
                        let list = Alpine.raw(this.list);
                        console.log(list);
                        let moved_step = this.list.splice(event.oldIndex, 1)[0]
                        this.list.splice(event.newIndex, 0, moved_step)
                    },
                });
            }, 100);

        },
        editModule: (curriculumId, moduleId) => {
            this.editingModule = `curriculum_` + curriculumId + `_module_` + moduleId;
            document.getElementById(`module_` + moduleId).style.display = '-webkit-inline-box';
            console.log(this.editingModule);
        },
        shouldShowEditField: (curriculumId, moduleId) => {
            console.log(this.editingModule);
            return this.editingModule === `curriculum_` + curriculumId + `_module_` + moduleId;
        },
    }
}
`

**Comment by danharrin** [comment]
Send the PHP for this field please
